## 13.9 Interaction between two continuous predictor variables

We have seen that when $X_1$ is continuous and $X_2$ is binary, the interaction term is interpretable as the difference in slopes ($Y$ on $X_1$) between two groups defined by the binary variable $X_2$. When both $X_1$ and $X_2$ are continuous, the interaction term is interpretable as the difference in slopes ($Y$ on $X_1$) per unit increase in $X_2$.

*Example:* To illustrate continuous-continuous interactions, we will fit a model relating birthweight ($Y$) to length of pregnancy ($X_1$), mother's height ($X_2$; $X_2$ is now treated as a continuous variable) and the interaction between length of pregnancy and mother's height ($X_3$).

In [7]:
#Create the interaction term 
data$int3<-data$Gestational.Days*data$Maternal.Height

#Include the interaction term in the regression model 
int_model3<-lm(data$Birth.Weight~data$Gestational.Days+data$Maternal.Height+data$int3)
summary(int_model3)


Call:
lm(formula = data$Birth.Weight ~ data$Gestational.Days + data$Maternal.Height + 
    data$int3)

Residuals:
    Min      1Q  Median      3Q     Max 
-53.794 -10.572   0.204  10.265  54.515 

Coefficients:
                        Estimate Std. Error t value Pr(>|t|)
(Intercept)           -27.200984 222.960540  -0.122    0.903
data$Gestational.Days   0.232172   0.799566   0.290    0.772
data$Maternal.Height    0.320110   3.473652   0.092    0.927
data$int3               0.003432   0.012453   0.276    0.783

Residual standard error: 16.45 on 1170 degrees of freedom
Multiple R-squared:  0.1969,	Adjusted R-squared:  0.1949 
F-statistic: 95.63 on 3 and 1170 DF,  p-value: < 2.2e-16


To interpret the above output, let's first focus on the relationship between birthweight and height in babies born at 200 gestational days. 

\begin{align}
E(Y|X_1=200)&=-27.201+0.232 \times 200+0.320 \times X_2+0.003 \times 200 \times X_2\\
 &= 19.200 + 0.920X_2
\end{align}

Amongst babies born at 201 days the analogous relationship is:

\begin{align}
E(Y|X_1=201)&=-27.201+0.232 \times 201+0.320 \times X_2+0.003 \times 201 \times X_2\\
 &=  19.431 + 0.923X_2
\end{align}

Amongst babies born at 202 days the analogous relationship is:

\begin{align}
E(Y|X_1=202)&=-27.201+0.232 \times 202+0.320 \times X_2+0.003 \times 202 \times X_2\\
 &=  19.662 + 0.926X_2
\end{align}

The interpretation is that at each number of gestational days, the relationship between height and birthweight is linear with the slope of association increasing by 0.003 ounces ($\hat{\beta_3}$) for each one day increase. However, in this example the $p$-value associated with the estimated regression coefficient for the interaction term is 0.783, indicating that the data do not provide evidence of an interaction between gestational days and mother's height on birthweight. 


